.github/workflows/olm_scorecard.yml

```diff
@@ -0,0 +1,28 @@
+name: Operator CI
+
+on: [push, pull_request]
+
+jobs:
+  scorecard-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      
+    - name: Install Operator SDK
+      run: |
+        curl -LO https://github.com/operator-framework/operator-sdk/releases/download/v1.24.1/operator-sdk_linux_amd64
+        chmod +x operator-sdk_linux_amd64
+        sudo mv operator-sdk_linux_amd64 /usr/local/bin/operator-sdk
+
+    - name: Install yq
+      run: |
+        sudo wget -O /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_amd64
+        sudo chmod +x /usr/bin/yq
+
+    - name: Modify scorecard config
+      run: |
+        yq e 'del(.stages[].tests[] | select(.labels.test == "olm-crds-have-resources-test"))' -i bundle/tests/scorecard/config.yaml
```

Comment:
What is this doing exactly?

Reply:
it removes this [known bug](https://github.com/operator-framework/operator-sdk/issues/5269) which fails